lib/tds/types.ex

```diff
@@ -358,27 +358,32 @@ defmodule Tds.Types do
 
         {type_info, rest}
 
-      data_type_code == @tds_data_type_image ->
-        # TODO NumBarts Reader
-        <<length::signed-32, numparts::signed-8, rest::binary>> = tail
+      user_type == @tds_data_type_image ->
+        # TODO NumParts Reader
+        <<length::little-unsigned-32, numparts::signed-8, rest::binary>> = tail
+        IO.puts("image length: #{length}")
+        IO.puts("image numparts: #{numparts}")
```

Comment:
Did you mean to leave in these `IO.puts` statements?

Reply:
I removed them in other commit in master